scripts/dream.py

```diff
@@ -458,7 +458,17 @@ def prepare_image_metadata(
     if postprocessed and opt.save_original:
         filename = choose_postprocess_name(opt,prefix,seed)
     else:
-        filename = f'{prefix}.{seed}.png'
+        wildcards = dict(opt.__dict__)
```

Comment:
All looks good. When I merge this (today or tomorrow) I'll just move this part of the `ifelse` into a new function in order to avoid code bloat.